File: september/13-09/13-09.py
def find_missing_numbers(arr):

    big_number = max(arr)
    missing_numbers = []

    for i in range(big_number):
        number = i+1
        if number not in arr:
            missing_numbers.append(number)

    return missing_numbers


find_missing_numbers([1, 2, 3, 4, 5]) # should return [].
find_missing_numbers([10, 1, 10, 1, 10, 1]) #should return [2, 3, 4, 5, 6, 7, 8, 9].

File: september/13-09/13-09.js
function findMissingNumbers(arr) {

    const bigNumber = Math.max(...arr);
    let missingNumbers = [];

    for(let i = 1; i <= bigNumber; i++){
        if(!arr.includes(i)){
        missingNumbers.push(i);
        }
    }

    return missingNumbers;
}